src/System.Private.ServiceModel/tools/setupfiles/CleanupWCFTestService.cmd

```diff
@@ -1,20 +1,11 @@
 echo off
 setlocal
 
-echo BridgeKeepRunning=%BridgeKeepRunning%
-if '%BridgeKeepRunning%' neq 'true' (
-    echo Releasing Bridge resources and stopping it if running locally.
-    pushd %~dp0..\..\..\..\bin\wcf\tools\Bridge
-    echo Invoking Bridge.exe -stopIfLocal -reset %* ...
-    call Bridge.exe -stopIfLocal -reset %*
-    popd
-) else (
-    echo Releasing Bridge resources but leaving it running.
-    pushd %~dp0..\..\..\..\bin\wcf\tools\Bridge
-    echo Invoking Bridge.exe -reset %* ...
-    call Bridge.exe -reset %*
-    popd
-    echo The Bridge was left running because BridgeKeepRunning is true
-)
+echo Releasing Bridge resources and stopping it if running locally.
+pushd %~dp0..\..\..\..\bin\wcf\tools\Bridge
+echo Invoking Bridge.exe -stopIfAutoStart -reset %* ...
+call Bridge.exe -stopIfAutoStart -reset %*
```

Comment:
So I see two pivots based on this discussion: 

* The Bridge should (not) be allowed to be terminated by test script
* The Bridge should (not) shut down after a timeout occurs (use case: when running locally, i *never* want the bridge to shut down by itself, but on CI boxes, shut them down after x hours to rescue the machines. This can be achieved by setting the timeout to be infinite).

Perhaps "AutoShutdown" or "ShutdownAfterTest" might be a more useful moniker for StartBridge/EnsureBridge, given the different modes seem to be talking about how we intend to *stop* the bridge, rather than how we intend to *start* it.

Aside: FWIW, I personally always found I needed to do a double-take when thinking about "StartBridge" and "EnsureBridge" and how it interacts with timeouts.